koku/api/report/queries.py

```diff
@@ -48,6 +48,12 @@ def strip_tag_prefix(tag):
     return tag.replace("tag:", "").replace("and:", "").replace("or:", "")
 
 
+def is_grouped_by_tag(parameters):
+    """Determine if grouped by tag."""
+    group_by = list(parameters.parameters.get("group_by", {}).keys())
+    return [key for key in group_by if "tag" in key]
```

Comment:
```suggestion
    return any(["tag" in key for key in group_by])
```

Reply:
Totally optional, as its use will be the same, but this will return a single boolean if we have any tag group bys.

Reply:
I actually use the return later on to get what the `tag_key` is for when the tag doesn't exist to create the `no-{tag_key}` value.